app/kumactl/cmd/get/testdata/get-fault-injection.golden.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I still see some separators. Can you please check all the YAML files that you add in this PR?